proto/zitadel/oidc/v2alpha/authorization.proto

```diff
@@ -0,0 +1,117 @@
+syntax = "proto3";
+
+package zitadel.oidc.v2alpha;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+option go_package = "github.com/zitadel/zitadel/pkg/grpc/oidc/v2alpha;oidc";
+
+message AuthRequest{
+  option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
+    external_docs: {
+      url: "https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest";
+      description: "Find out more about OIDC Auth Request parameters";
+    }
+  };
```

Comment:
This top-level schema documentation does not get rendered somehow:
https://docs-f7o3cd32k-zitadel.vercel.app/docs/apis/resources/oidc_service/oidc-service-get-auth-request

But it is valid syntax as per this example:

https://github.com/grpc-ecosystem/grpc-gateway/blob/06fae32b118ceb06e71790117a41c323f70de03e/examples/internal/proto/examplepb/a_bit_of_everything.proto#L235-L238

Is it a bug or should this be handeled differently?

Reply:
Hm on a first glance it looks correct.

Let me check if the generated output contains the url.